users/api/http/endpoint.go

```diff
@@ -23,6 +24,55 @@ func registrationEndpoint(svc users.Service) endpoint.Endpoint {
 	}
 }
 
+// Password reset endpoint serves post request with email of the user
+// for whom password reset flow is to be initiated.
+// If request is successful email with reset link will be sent to the
+// email specified in the request.
+// Link contains token that has TTL that needs to be verified.
+// When user gets email with reset password link that hits this endpoint
+// which will return response into the ui form which then can
+// be used to enter email and new password and then request can be submitted
+// to the post endpoint which actually changes password.
+func passwordResetRequestEndpoint(svc users.Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(passwResetReq)
+		res := resetPassRes{}
```

Comment:
Replace `Pass` with `Passwd`. It was uncler to me reading the code is the `pass` referencing to passing something...